agent/app/agent_unix.go

```diff
@@ -148,13 +149,18 @@ func isInstanceLaunchedInVPC(err error) bool {
 // a. ecs-eni
 // b. ecs-bridge
 // c. ecs-ipam
+// d. aws-appmesh
 func (agent *ecsAgent) verifyCNIPluginsCapabilities() error {
 	// Check if we can get capabilities from each plugin
 	for _, plugin := range awsVPCCNIPlugins {
 		capabilities, err := agent.cniClient.Capabilities(plugin)
 		if err != nil {
 			return err
 		}
+		// appmesh plugin does not contain aws vpc network capability
```

Comment:
I think a more appropriate comment would be: `// appmesh plugin is not needed for awsvpc networking capability`

Reply:
updated